packages/components/src/base-control/index.js

```diff
@@ -3,11 +3,11 @@
  */
 import classnames from 'classnames';
 
-function BaseControl( { id, label, help, className, children } ) {
+function BaseControl( { id, label, hideLabel, help, className, children } ) {
```

Comment:
`hideLabel` - this new prop has to be documented. I'm a bit afraid that the name proposed when read outside of its implementation might be confusing. Based on: https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/:

> The screen-reader-text class is used:
> * to hide text from vision that is only useful for screen readers users.
> * for skip links for keyboard navigation before the link gets focus.

It should be something like `hideLabelFromVision`.